backend/corpora/dataset_processing/make_seurat.R

```diff
@@ -5,4 +5,9 @@ install_version("SeuratObject", version = "4.0.2", repos = "http://cran.rstudio.
 
 h5adPath <- commandArgs(trailingOnly = TRUE)[1]
 
-sceasy::convertFormat(h5adPath, from="anndata", to="seurat", outFile = gsub(".h5ad", ".rds", h5adPath), main_layer = "data")
+sceasy::convertFormat(h5adPath,
+                      from="anndata",
+                      to="seurat",
+                      outFile = gsub(".h5ad", ".rds", h5adPath),
+                      main_layer = "data",
+                      target_uns_keys = c("schema_version", "title", "X_normalization", "batch_condition", "default_embedding", "X_approximate_distribution"))
```

Comment:
The forked repo adds a new optional parameter that specifies the list of keys from the AnnData object's `uns` metadata to migrate to the Seurat object's `misc` slot.

The allowed keys are specified in the curation schemas:
* https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/schema.md#uns-dataset-metadata
* https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/2.0.0/seurat_encoding.md#dataset-metadata

Note, the latter will be updated to reflect the above when this PR is merged:
* https://github.com/chanzuckerberg/single-cell-curation/pull/179